src/components/cdPipeline/cdPipeline.types.ts

```diff
@@ -158,12 +159,18 @@ interface ValidationPayloadApplicationMetaDataType {
     applicationObjectNamespace: string
 }
 
+interface ValidationPayloadFluxReleaseMetaDataType {
```

Comment:
Can be renamed to `ReleaseMetaDataType` and reused in `helmReleaseMetadata` and `fluxReleaseMetadata`